src/pages/_app.js

```diff
@@ -33,6 +33,10 @@ import env from '../lib/env';
 import isCordova from '../lib/isCordova';
 
 let isServer = false;
+// only used in serverSideRendering when getting the initial props
+// used for storing the initial props instead of serializing them for the client
+// to prevent sending too large html chunks, that break e.g. twitter cards
+let nonSerializedProps: ?{ appProps: AppProps, routeProps: any | void } = null;
```

Comment:
0: If it's hardcoded anyway: How about `propsOmittedForTwitterBot`?

Reply:
I assume we could also just this for others only interested in the head, so I went for a more generic name.